sources/engine/Stride.Assets/StrideConfig.cs

```diff
@@ -253,36 +253,6 @@ internal static bool IsVSComponentAvailableAnyVersion(IDictionary<Version, strin
             return false;
         }
 
-        /// <summary>
```

Comment:
This code is no longer used in the code base and is incompatible with these changes so just removing it.